src/it/projects/mexec-gh-389-block-exit-non-zero/verify.groovy

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
```

Comment:
There is no AFS project ...., we don't have a `NOTICE` file ...
so my proposition is to put standard header for new files, like:

```
Copyright MojoHaus and Contributors

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
```

Reply:
@kriegaex - If you as author of file has not an objection I can update it before release.

Reply:
Sure, no problem. Please note that I copied another IT when starting to create my 3 new ones. That means, the same faulty header must be in at least one more file, probably many. You may want to update them all.

Reply:
I should not change by replace all licenses headers.

We need to discover history for all files ... maybe some one comes from ASF product ... and in such case should be preserved.
So it can be a huge work which I wnant not to do now.

For new files it is clear.

Reply:
Replaced all copyright headers in the two new plugin classes and the three groovy IT verification scripts. @slawekjaranowski, is there anything else stopping you from merging? I want to keep the difference between touch time and cycle time for this PR as low as possible, because frequent context switches are just bad for productivity.